### ISSUE TITLE
Question regarding binary time series classification 

### ISSUE THREAD

**Issue by youssefmecky96** [pytorch/captum#1201]
I want to use Integrated gradients to attribute to time steps in the inputs. The output of my model is in the shape (Batch_size,Time steps, Class) .My labels would be of size ((Batch_size,Time steps, Class))  would be of size however  I am not sure how to use the target variable in the attribution as it seems I need to choose a time step and a target class to attribute like below. 
Maybe I misunderstood the documentation somewhere ? 
```
attr, delta = explantation.attribute(
                data, target=(22, 1), return_convergence_delta=True, baselines=baselines, n_steps=20
            )
```

**Comment by craymichael** [comment]
Apologies for the delay.

For an output shaped like:

```text
[batch_size, time_steps, num_classes]
```

the Captum target for one scalar output should be:

```python
target = (time_index, class_index)
```

So this:

```python
target=(22, 1)
```

means "attribute class 1 at output time step 22."

If you want an attribution for every time step's class prediction, you have two options:

1. Loop over time steps:

```python
attrs = []
for t in range(num_time_steps):
    attr = ig.attribute(data, baselines=baselines, target=(t, class_idx))
    attrs.append(attr)
```

2. Write a forward wrapper that reduces the output to the scalar you care about, for example the sum of the predicted class scores across all time steps:

```python
def forward_sum_selected(inputs):
    logits = model(inputs)  # [B, T, C]
    return logits[:, :, class_idx].sum(dim=1)
```

Use the loop if you need separate explanations for each output time step. Use the summed wrapper if you want one explanation for the aggregate sequence-level objective.

This should answer the question, so I'll close this for now. Please feel free to reply with any follow-up questions.
